### PR TITLE
Tweaks for Bullseye

### PR DESCRIPTION
**Pull request by JedMeister**
Some minor (but required) changes for Bullseye.

### CODE REVIEW

**Comment by JedMeister** [comment]
@OnGle

File: octohub/connection.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def send(
         if params is None or params == {}:
             params = dict()
 
-        url = self.endpoint + uri
+        url = self.endpoint + str(uri)
```
Comment (OnGle) [inline octohub/connection.py:82]: Argument typing suggests uri is already a string, if it is not, the signature should be changed.

**Review by OnGle — CHANGES_REQUESTED** [review]
Minor change requuested.

File: octohub/connection.py (hunk 1 above)
Comment (JedMeister) [reply octohub/connection.py:82]: Yep good call. I vaguely recall adding that and not being sure if it was needed or not, but figured it doesn't hurt. But now you mention it and I look at it in full context, you are right! I'll just remove it.

**Pull request merged**